src/splitInteger.test.js

```diff
@@ -1,21 +1,24 @@
-'use strict';
+"use strict";
 
-const splitInteger = require('./splitInteger');
-
-test(`should split a number into equal parts
-  if a value is divisible by a numberOfParts`, () => {
+const splitInteger = require("./splitInteger");
 
+test(`should split a number into equal parts if a value is divisible by numberOfParts`, () => {
+  expect(splitInteger(8, 4)).toEqual([2, 2, 2, 2]);
+  expect(splitInteger(12, 3)).toEqual([4, 4, 4]);
 });
 
-test(`should return a part equals to a value
-  when splitting into 1 part`, () => {
-
+test(`should return a part equal to the value when splitting into 1 part`, () => {
+  expect(splitInteger(6, 1)).toEqual([6]);
```

Comment:
There's no check for the function's behavior when splitting into 1 part when the value is not a single digit. A test for this scenario is recommended to ensure that the function behaves as expected.